grails-app/domain/org/pih/warehouse/order/Order.groovy

```diff
@@ -21,31 +21,37 @@ import org.pih.warehouse.shipping.ShipmentStatusCode
 
 class Order implements Serializable {
 
-    def beforeInsert = {
+    def beforeInsert() {
```

Comment:
closures for events work also in Grails 3, but just the code is "gray", so since I was "touching" this code while debugging, I decided to change it to the methods, to follow the Grails 3 convention

Reply:
If I understand correctly this change to closures doesn't affect the functionality of the event and you changed it because it is a preferable way to do it in grails 3 and not because it fixes something, right?

Reply:
@drodzewicz yes, exactly. It is working, but if we take a look at grails 3 docs, they (event) are specified as methods:
https://docs.grails.org/3.0.x/guide/GORM.html#eventsAutoTimestamping